### PR TITLE
Carousel list

### PR DESCRIPTION
**Pull request by sureshhemal**

### CODE REVIEW

**Review by sureshhemal — COMMENTED** [review]
@Kushan1994  please review

File: resources/js/Pages/Backend/Carousel/Index.vue [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+        type: Array,
+        required: true,
+    },
+})
 
 </script>
 
 <template>
-    Carousel List
+    <ul role="list" class="grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-3 sm:gap-x-6 lg:grid-cols-4 xl:gap-x-8">
```
Comment (Kushan1994) [inline resources/js/Pages/Backend/Carousel/Index.vue:12]: grid 1a use wenawad

Comment (sureshhemal) [reply resources/js/Pages/Backend/Carousel/Index.vue:12]: I think so.. pull

Comment (Kushan1994) [reply resources/js/Pages/Backend/Carousel/Index.vue:12]: pull kiwwe

Comment (sureshhemal) [reply resources/js/Pages/Backend/Carousel/Index.vue:12]: pull karan balanna

**Pull request merged**